### PR TITLE
Update yaspeller to latest version

### PR DESCRIPTION
**Pull request by mquhuy**
We are using way too old yaspeller version, which caused issue #378

### CODE REVIEW

**Comment by tuminoid** [comment]
Cool finding. Now it has some whitespace issue tho, it is suggesting to fix almost every word with " word" (ie. added leading space)... another config issue?

```console
Typos: 1
1. In (suggest:  In)
```

File: .yaspeller.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "maxRequests": 5,
   "ignoreDigits": true,
   "ignoreText": [["^layout:.*$", "m"]],
+  "ignoreTags": [["**", "__"]],
```
Comment (tuminoid) [inline .yaspeller.json:7]: Per https://github.com/hcodes/yaspeller readme, `ignoreText` and `ignoreTags` is supposed to be Array, not array of Arrays.

Comment (mquhuy) [reply .yaspeller.json]: Ok then

**Comment by tuminoid** [comment]
It seems to pass in CI (says "Passed" in th log), but it seems to care which files are edited. So if I edit a readme to get yaspeller triggered, it still explodes.

```console
yaspeller................................................................Failed
- hook id: yaspeller
- exit code: 1

/home/tumi/.cache/pre-commit/repoz9kpkrkq/node_env-system/lib/node_modules/yaspeller/node_modules/minimatch/dist/cjs/ast.js:86
    #fillNegs() {
             ^

SyntaxError: Unexpected token '('
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/tumi/.cache/pre-commit/repoz9kpkrkq/node_env-system/lib/node_modules/yaspeller/node_modules/minimatch/dist/cjs/index.js:9:18)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
```

Does it do same for you, if you edit readme.md and then try commit it, or is it just on my system?

**Comment by mquhuy** [comment]
``

> It seems to pass in CI (says "Passed" in th log), but it seems to care which files are edited. So if I edit a readme to get yaspeller triggered, it still explodes.
> 
> ```
> yaspeller................................................................Failed
> - hook id: yaspeller
> - exit code: 1
> 
> /home/tumi/.cache/pre-commit/repoz9kpkrkq/node_env-system/lib/node_modules/yaspeller/node_modules/minimatch/dist/cjs/ast.js:86
>     #fillNegs() {
>              ^
> 
> SyntaxError: Unexpected token '('
>     at wrapSafe (internal/modules/cjs/loader.js:915:16)
>     at Module._compile (internal/modules/cjs/loader.js:963:27)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
>     at Module.load (internal/modules/cjs/loader.js:863:32)
>     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
>     at Module.require (internal/modules/cjs/loader.js:887:19)
>     at require (internal/modules/cjs/helpers.js:74:18)
>     at Object.<anonymous> (/home/tumi/.cache/pre-commit/repoz9kpkrkq/node_env-system/lib/node_modules/yaspeller/node_modules/minimatch/dist/cjs/index.js:9:18)
>     at Module._compile (internal/modules/cjs/loader.js:999:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
> ```
> 
> Does it do same for you, if you edit readme.md and then try commit it, or is it just on my system?

I made a trial in this PR: I added a line in README, but the test still passed. It passed for me locally as well.
Did you do something more special in your edit?

**Review by tuminoid — COMMENTED** [review]
Must be some glitch in my env then.

/lgtm

**Review by lentzi90 — COMMENTED** [review]
/approve

**Pull request merged**